static/js/components/inputs/inputs_test.js

```diff
@@ -39,13 +40,15 @@ describe('Profile inputs', () => {
       {value: 'o', label: 'Other/Prefer not to say'}
     ];
 
-    let renderGenderSelectField = () => {
-      Object.assign(inputProps, {
-        keySet: ['gender'],
-        label: "Gender",
-        options: genderOptions
-      });
-      return renderTestSelectField();
+    let renderGenderSelectField = (props = {}) => {
+      return renderTestSelectField(
+        Object.assign(inputProps, {
```

Comment:
Can you change this line to `Object.assign({}, inputProps, {` so `inputProps` isn't altered?